sandbox-templates/mdx-evaluate-create-react-app/package.json

```diff
@@ -7,11 +7,11 @@
     "@mdx-js/mdx": "2.0.0-rc.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-scripts": "5.0.0-next.58"
+    "react-scripts": "5.0.0"
   },
   "devDependencies": {
-    "@babel/runtime": "7.16.3",
-    "typescript": "4.5.2"
+    "@babel/runtime": "7.16.7",
+    "typescript": "4.4.4"
```

Comment:
keeping this version pinned until https://github.com/microsoft/TypeScript/issues/46900 is resolved